routers/api/v1/repo/issue_label.go

```diff
@@ -310,9 +341,43 @@ func prepareForReplaceOrAdd(ctx *context.APIContext, form api.IssueLabelsOption)
 		return
 	}
 
-	labels, err = models.GetLabelsByIDs(form.Labels)
-	if err != nil {
-		ctx.Error(http.StatusInternalServerError, "GetLabelsByIDs", err)
+	switch form.Mode {
+	case "name_only":
```

Comment:
Do these values really need to be different than `name`, `org_name`, `id`, as already used above?
If yes, these values should be documented in swagger